packages/back-end/src/models/OrganizationModel.ts

```diff
@@ -405,3 +407,38 @@ export function toOrganizationApiInterface(
     dateCreated: dateCreated?.toISOString() || "",
   };
 }
+
+export async function updateMember(
+  orgId: string,
```

Comment:
Take the full OrganizationInterface as an argument instead of orgId.  Avoids an unnecessary db lookup and simplifies this function a bit.